### PR TITLE
Add API tests

### PR DESCRIPTION
**Pull request by michplunkett**
## What does this pull request add?
Added test cases for the `api` layer.

## Are there any technical things that should be noted for this PR?
`make test` should actually do something now.

## How was it tested?
- [x] Wrote test cases and they pass.
```
======================================== test session starts ========================================
platform darwin -- Python 3.9.7, pytest-6.2.4, py-1.10.0, pluggy-0.13.1 -- /opt/anaconda3/bin/python
cachedir: .pytest_cache
rootdir: /Users/shaymilner/Documents/Harris/01Winter23/CAPP_II/final_project/reproductive-rights-data-project
plugins: anyio-2.2.0
collected 5 items                                                                                   

test/test_api.py::test_get_api_data PASSED
test/test_api.py::test_to_json Writing out to json file
PASSED
test/test_api.py::test_set_default_types PASSED
test/test_api.py::test_fill_in_missing_data PASSED
test/test_api.py::test_add_missing_states PASSED

========================================= 5 passed in 0.14s =========================================
```

- [x] I ran `pylint` and have made the bulk of changes that it has requested.
- [x] I ran the `black` formatter and pushed the changes.

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -4,5 +4,5 @@
 *py @michplunkett @ehabich
 
 ./visualization @necabotheking @michplunkett
-./apis @chanteriam @necabotheking
+./api @chanteriam @necabotheking
```
Comment (michplunkett) [inline CODEOWNERS:7]: `apis` is now `api`.

File: visualizations/vis_class.py [hunk 2]
```diff
@@ -4,7 +4,7 @@
 
 class Visualization(MutableMapping):
     @abstractmethod
-    def __init__(self, filename):
+    def __init__(self, file_name):
```
Comment (michplunkett) [inline visualizations/vis_class.py:7]: Done did it.

File: test/test_api.py [hunk 3]
```diff
@@ -4,3 +4,271 @@
...
+    print("test")
+
+
+""" MICHAEL PLUNKETT CONTRIBUTION ABOVE """
+
+
+""" CHANTERIA MILNER CONTRIBUTION BELOW """
+
+
+@responses.activate
```
Comment (michplunkett) [inline test/test_api.py:185]: You don't need this line since you aren't using the responses package here.

**Review by ehabich — APPROVED** [review]

**Review by necabotheking — COMMENTED** [review]
Looks good!

**Pull request merged**